go.mod

```diff
@@ -15,35 +15,34 @@ require (
 	github.com/operator-framework/helm-operator-plugins v0.7.0
 	github.com/operator-framework/operator-controller v1.0.0-rc1
 	github.com/operator-framework/operator-registry v1.48.0
-	github.com/spf13/cobra v1.8.1
-	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.9.0
-	golang.org/x/net v0.30.0
-	golang.org/x/text v0.19.0
-	helm.sh/helm/v3 v3.16.2
-	k8s.io/api v0.32.2
-	k8s.io/apiextensions-apiserver v0.32.2
-	k8s.io/apimachinery v0.32.2
-	k8s.io/client-go v0.32.2
-	k8s.io/component-base v0.32.2
+	github.com/spf13/cobra v1.9.1
+	github.com/spf13/pflag v1.0.6
+	github.com/stretchr/testify v1.10.0
+	golang.org/x/net v0.40.0
+	golang.org/x/text v0.26.0
+	helm.sh/helm/v3 v3.18.4
+	k8s.io/api v0.33.2
+	k8s.io/apiextensions-apiserver v0.33.2
+	k8s.io/apimachinery v0.33.2
+	k8s.io/client-go v0.33.2
+	k8s.io/component-base v0.33.2
```

Comment:
Are those the identical versions used in upstream?

Reply:
We don't have an upstream cluster-olm operator, and this looks like it's aligned with https://github.com/openshift/operator-framework-operator-controller/blob/main/go.mod#L36-L40 so I think we're good here.